platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/style/StyleFileActivity.java

```diff
@@ -115,6 +155,28 @@ private void writeToFile(File file, String contents) throws IOException {
     }
   }
 
+  static class RawResourceReaderWriter {
```

Comment:
`readRawResource` is duplicated across different activities (i.e. `QueryRenderedFeaturesBoxSymbolCountActivity`, `DataDrivenStyleActivity`, `RuntimeStyleActivity` and `GeoJsonSourceTests`). We could extract `RawResourceReaderWriter` and remove that duplication.